src/library/scala/math/BigInt.scala

```diff
@@ -178,19 +183,19 @@ final class BigInt(val bigInteger: BigInteger) extends ScalaNumber with ScalaNum
 
   /** Less-than-or-equals comparison of BigInts
    */
-  def <= (that: BigInt): Boolean = compare(that) <= 0
+  override def <= (that: BigInt): Boolean = compare(that) <= 0
```

Comment:
Again, the symbolic comparison operators are all defined in Ordered, and they're defined exactly like this.  Do we need to override them at all, instead of just deleting them?